src/components/Calculator.js

```diff
@@ -1,72 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
 import Result from './Result';
 import InputValue from './InputValue';
 import calculate from '../logic/calculate';
 
-// eslint-disable-next-line react/prefer-stateless-function
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      obj: {
-        total: null,
-        next: null,
-        operation: null,
-      },
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
+const Calculator = () => {
+  const [obj, setObj] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
```

Comment:
- [ ] When I run the app I see only empty white space on the screen, there is no calculator. Please fix this issue and make sure the calculator is visible on the screen and previous functionalities are maintained.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/170481399-13d2288d-a17b-4984-9587-578a3e770c61.png)

Reply:
@thecodechaser 
Thank you for identifying the bug, I have now fixed the problem